apps/playground/src/app/modules/components/stepper-module/stepper-module.component.ts

```diff
@@ -2,10 +2,19 @@
  * Angular imports
  */
 import { Component } from '@angular/core';
+import { NonNullableFormBuilder } from '@angular/forms';
 
 @Component({
   templateUrl: './stepper-module.component.html',
   styleUrls: ['./stepper-module.component.scss']
 })
 export class StepperModuleComponent {
+  formStepper = this.formBuilder.group({
+    stepTitle: 'The step\'s title',
+    stepNumber: 1,
+    numberOfSteps: 3,
+    nextStepTitle: 'The next step\'s title'
+  });
```

Comment:
You need to add `Validators` for the fields that are required + the handling of displaying an error message when these fields are invalid.

I suggest you make sure you have the latest version of develop on this branch and then see what I've done in `apps/playground/src/app/modules/components/tab-module/tab-module.component.ts` to handle that.

You'll probably need also to move the handling of missing `Input` properties inside the `DsfrStepperComponent` within `ngOnChanges`. Finally, don't forget to use the latest `required: true` option for `Input` properties that came with Angular 16 😉

Reply:
Error messages are still not displayed...

Look at how it's done in the `TabModuleComponent`...